lib/rspec/buildkite/analytics/session.rb

```diff
@@ -50,20 +28,18 @@ def initialize(url, authorization_header, channel)
       @url = url
       @authorization_header = authorization_header
 
-      @logger = Logger.new
-
       reconnection_count = 0
 
       begin
         reconnection_count += 1
         connect
       rescue TimeoutError, InitialConnectionFailure => e
-        @logger.write("rspec-buildkite-analytics could not establish an initial connection with Buildkite due to #{e}. Attempting retry #{reconnection_count} of #{MAX_RECONNECTION_ATTEMPTS}...")
+        RSpec::Buildkite::Analytics.logger.error("rspec-buildkite-analytics could not establish an initial connection with Buildkite due to #{e}. Attempting retry #{reconnection_count} of #{MAX_RECONNECTION_ATTEMPTS}...")
         if reconnection_count > MAX_RECONNECTION_ATTEMPTS
-          $stderr.puts "rspec-buildkite-analytics could not establish an initial connection with Buildkite due to #{e.message} after #{MAX_RECONNECTION_ATTEMPTS} attempts. You may be missing some data for this test suite, please contact support if this issue persists."
+          RSpec::Buildkite::Analytics.logger.debug "rspec-buildkite-analytics could not establish an initial connection with Buildkite due to #{e.message} after #{MAX_RECONNECTION_ATTEMPTS} attempts. You may be missing some data for this test suite, please contact support if this issue persists."
```

Comment:
Should this message be logged as an `error` given the potential severity of missing data?

Reply:
Agreed. This is an error imo.